planner/core/show_predicate_extractor.go

```diff
@@ -138,3 +139,46 @@ func (e *ShowTablesTableExtractor) explainInfo() string {
 	}
 	return s
 }
+
+// ShowVariablesExtractor is used to extract some predicates of variables.
+type ShowVariablesExtractor struct {
+	ShowColumnsTableExtractor
```

Comment:
```Extract``` is the same one as the ```ShowTablesTableExtractor```. Can we Inherit it from the parent class? this is that parent class we create a new one and call it ```BaseExtractor```.

Reply:
OK,  I'll learn and check, THKS.

Reply:
I made changes to be unified ShowBaseExtrator format, please check, THKS much.